ui/src/testing/stubs/resources-stub.ts

```diff
@@ -0,0 +1,53 @@
+import {WorkspaceData} from 'app/utils/workspace-data';
+import {
+    BillingStatus,
+    Cohort,
+    CohortReview,
+    ConceptSet,
+    DataSet,
+    FileDetail,
+    ResourceType,
+    WorkspaceAccessLevel,
+    WorkspaceResource,
+} from 'generated/fetch';
+import {CdrVersionsStubVariables} from 'testing/stubs/cdr-versions-api-stub';
+import {WorkspaceStubVariables} from 'testing/stubs/workspace-service-stub';
+
+type InputResource = FileDetail | Cohort | CohortReview | ConceptSet | DataSet;
+
+export function convertToResource(
```

Comment:
Can this be combined with the real implementation above? It looks like this function is a superset of the functionality that's in the notebook file.

Reply:
yes - the original reason for separating these is no longer valid